api/app/auto_spatial_advisory/sfms.py

```diff
@@ -0,0 +1,83 @@
+""" Code related to managing SFMS output files.
+"""
+import os
+from datetime import date
+from app.schemas.auto_spatial_advisory import SFMSFile, SFMSRunType
+from app.utils.time import get_hour_20, get_vancouver_now
+
+
+def get_date_part(filename: str) -> str:
+    """ Get the date part of the filename.
+    Filename example: hfi20220823.tif
+    """
+    return filename[filename.rfind('.') - 8:filename.rfind('.')]
+
+
+def is_actual(filename: str) -> bool:
```

Comment:
Minor: could simplify this to return `actual` or `forecast` and kill the `get_prefix` function to simplify